### ISSUE TITLE
Add a WebServerSender

### ISSUE THREAD

**Issue by sfehlandt** [IntegratedAlarmSystem-Group/ias#41]
We need a module that reads Alarms from the Kafka Queue and sends them through websockets to the IAS Webserver.
It should also filter non-alarms (at least in the meantime)

**Comment by sfehlandt** [comment]
Closed by a283fbf9224ca2cd0790230b6ddf684f0c6848be

**Comment by acaproni** [comment]
Nightly test failed in [jenkins](http://iastest.hq.eso.org:8080/job/IAS-develop-test/117/console):
```
1) testWebServerSender(org.eso.ias.webserversender.WebServerSenderTest)
03:59:46      [exec] java.lang.AssertionError: Some messages were not sent by the WebServerSender expected:<6> but was:<4>
03:59:46      [exec] 	at org.junit.Assert.fail(Assert.java:88)
03:59:46      [exec] 	at org.junit.Assert.failNotEquals(Assert.java:834)
03:59:46      [exec] 	at org.junit.Assert.assertEquals(Assert.java:645)
03:59:46      [exec] 	at org.eso.ias.webserversender.WebServerSenderTest.testWebServerSender(WebServerSenderTest.java:208)
```
Could you please check? Some events seem missing.

I have to check why jenkins did not detect the failure..

**Comment by sfehlandt** [comment]
The test does the following:
- Run a mock Server in a separate Thread
- Run a WebServerSender in a separate Thread and connect to the Server
- Publish messages iteratively in the Kafka queue

The WebServerSender should take these messages and send them to the Server.
We assert that the Server received all the messages.

Yesterday, addressing issue #47, we realised the test failed sometimes because somehow the Server missed the first couple messages. This was because sometimes the Server was not ready to process the messages when the WebServerSender started to send them. Therefore, the test had non-deterministic outcome with low probability of failure.

In the new version (for issue #47) we are introducing a verification in the test, so it starts to publish in the Kafka queue only after the WebServerSender receives a "Hello" message from the Server after connection. This should fix the issue.

This issue will be fixed when we close issue #47, which will be soon.

**Comment by sfehlandt** [comment]
Closed  by ac09750d6591b9bc998d09d5b7169d206bce2748
